contrib/native/client/src/protobuf/UserBitShared.pb.h

```diff
@@ -1,65 +1,152 @@
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
 // source: UserBitShared.proto
 
-#ifndef PROTOBUF_UserBitShared_2eproto__INCLUDED
-#define PROTOBUF_UserBitShared_2eproto__INCLUDED
+#ifndef PROTOBUF_INCLUDED_UserBitShared_2eproto
+#define PROTOBUF_INCLUDED_UserBitShared_2eproto
 
 #include <string>
 
 #include <google/protobuf/stubs/common.h>
 
-#if GOOGLE_PROTOBUF_VERSION < 2005000
+#if GOOGLE_PROTOBUF_VERSION < 3006001
```

Comment:
Please install 2.5 protobuf version. Drill currently uses it.
But it will be updated soon, see #1639

Reply:
@vdiravka 
I updated with the correct version (I think) of the protobufs.  Just as an FYSA (and in case I have to do this again), I was having a lot of trouble because I have Anaconda installed on my Mac which seems to install its own version of clang which doesn't work with these libraries.  Running with `sudo` permissions seemed to fix it.

```
sudo cmake -G "Unix Makefiles" -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl" -DOPENSSL_INCLUDE_DIR="/usr/local/opt/openssl/include" -DOPENSSL_LIBRARIES="/usr/local/opt/openssl/lib " ..
```